pages/docs/docs-style-guide.mdx

```diff
@@ -0,0 +1,114 @@
+
+2. Avoid using terms that have colonialist or racist connotations.
+
+   **Example:** 
+   - **Not this:** Contributors should push their pull requests to the **master** branch.
+   - **Use this:** Contributors should push their pull requests to the **main** branch.
+
+## Grammar & Mechanics
+
+### Commas
+
+1. Use commas when describing lists.
+
+   **Example:** 
+   - **Not this:** You can also add your **timeline testimonials and upcoming events** that you are participating in.
+   - **Use this:** You can also add your **timeline, testimonials, and upcoming events** that you are participating in.
+
+2. Use commas in introductory elements.
+
+   **Example:** 
+   - **Not this:** **If you do not have one yet** you can create one for free with an email address and password.
+   - **Use this:** **If you do not have one yet,** you can create one for free with an email address and password.
+
+3. Use commas before the seven coordinating conjunctions (and, but, so, for, nor, yet, or) to separate two independent clauses.
+
+   **Example:** 
+   - **Not this:** There are 4 ways you can add your profile **but** for this Quickstart we will use the GitHub UI.
+   - **Use this:** There are 4 ways you can add your profile, **but** for this Quickstart we will use the GitHub UI.
+
+### Capitalisation
+
+1. Only capitalise terms that describe a product, its feature, the first letter of a sentence, or a person's name.
+
+   **Example:** 
+    - **Not this:** There are 4 ways you can add your profile but for this **quickstart** we will use the **github ui**.
+    - **Use this:** There are 4 ways you can add your profile, but for this Quickstart we will use the **GitHub UI**.
+
+## Clarity
+
+### Active vs. Passive
+
+1. Remember how your writing instructor has often discouraged you from writing in the passive voice? Well, we need to make sure that contributors fully understand how to contribute to Linkfree, so it would be best to use active voice when working on our documentation.
+
+   **Example:** 
+    - **Not this:** Gitpod **was** an open-source application that provides a prebuilt development environment in your browser - powered by VS Code.
+    - **Use this:** Gitpod **is** an open-source application that provides a prebuilt development environment in your browser - powered by VS Code.
+
+### Jargon
+
+1. Since members of EddieHub tend to be new to open-source contributions, it would be best to use technical jargon sporadically so that they can gain a better understanding of how to contribute to Linkfree.
+
+   **Example:** 
+    - **Not this:** Gitpod **was** an open-source application that provides a prebuilt development environment in your browser - powered by VS Code.
+    - **Use this:** Gitpod **is** an open-source application that provides a prebuilt development environment in your browser - powered by VS Code.
+
+## Accessibility
+
+### Bullet points
+
+1. Avoid describing things only by their colour or position.
```

Comment:
What do you think of adding this here?
 ```suggestion
1. Avoid describing things by their color or position, instead use hyperlinks or figure numbers.
```

Reply:
> What do you think of adding this here?

The statement that occurs after the comment is ok. As far as changing "colour" to "color", Eddie and co wanted the Doc to be written in British English so I'll keep the term as is.